web-dev/src/components/LoginPage/index.js

```diff
@@ -1,111 +1,88 @@
-                                                           ref={passwordRef}
-                                                           autoComplete="on"
-                                                           placeholder="Password"
-                                                           className="login-user-input text-white"
-                                                           id="login-password"
-                                                           required
-                                                    />
-                                                </div>
-                                            </div>
-                                        </div>
-                                        <button onClick={handleSignin}
-                                            className="btn login-button btn-submit btn-small"
-                                                type="submit">
-                                            Sign In
-                                        </button>
-                                        <div className="mt-3 mb-3">
-                                            <Link to="/registration" className="login-registration-link text-white-50">
-                                                New to Netflicks?
-                                                <span className="text-white"> Sign up now. </span>
-                                            </Link>
-                                        </div>
-                                        <div className="mt-3 mb-3">
-                                            <Link to="/homepage" className="login-registration-link text-white-50">
-                                                Or
-                                                <span className="text-white"> Guest </span> for now.
-                                            </Link>
-                                        </div>
-                                    </form>
-                                </div>
-                            </div>
+  const emailRef = useRef();
+  const passwordRef = useRef();
+  const {signIn} = useAuth()
+  const handleSignin = event => {
+    signIn(emailRef.current.value, passwordRef.current.value).catch(() => {
+      alert("Invalid emailOrNumber/password");
+    });
+    event.preventDefault();
+  }
+  return (
+    <UnauthenticatedLock>
+      <div className="login-background flex-container"
+           style={{backgroundImage: `url(${LoginBackground})`}}>
+        <div className="login-bring-it-forward">
+          <img className="login-logo-size" src={require('./netflicks-logo.png')} alt="Logo" />
+          <div className="login-main-frame">
+            <div>
+              <div className="login-main-subframe">
+                <div className="login-main">
+                  <h1 className="login-title">Sign In</h1>
+                  <form onSubmit={handleSignin}>
```

Comment:
When we use form, we have to handle onSummit too. Otherwise, the browser will handle that in a weird way for you.